plugins/org.jboss.reddeer.eclipse/src/org/jboss/reddeer/eclipse/m2e/core/ui/preferences/MavenErrorPreferencePage.java

```diff
@@ -0,0 +1,130 @@
+/******************************************************************************* 
+ * Copyright (c) 2016 Red Hat, Inc. 
+ * Distributed under license by Red Hat, Inc. All rights reserved. 
+ * This program is made available under the terms of the 
+ * Eclipse Public License v1.0 which accompanies this distribution, 
+ * and is available at http://www.eclipse.org/legal/epl-v10.html 
+ * 
+ * Contributors: 
+ * Red Hat, Inc. - initial API and implementation 
+ ******************************************************************************/
+package org.jboss.reddeer.eclipse.m2e.core.ui.preferences;
+
+import org.jboss.reddeer.common.logging.Logger;
+import org.jboss.reddeer.jface.preference.PreferencePage;
+import org.jboss.reddeer.swt.impl.combo.LabeledCombo;
+
+/**
+ * Class representing "Maven->Errors/Warnings" preference page.
+ * 
+ * @author jkopriva, ldimaggi
+ *
+ */
+
+public class MavenErrorPreferencePage extends PreferencePage {
+
+	private static final String GROUP_ID = "\"groupId\" duplicate of parent groupId";
+	private static final String VERSION = "\"version\" duplicate of parent version";
+	private static final String PROJECT_CONFIG = "Out-of-date project configuration";
+	private static final String PLUGIN_EXECUTION = "Plugin execution not covered by lifecycle configuration";
+	private static final String MANAGED_VERSION = "Overriding managed version";
+	private static final Logger log = Logger.getLogger(MavenErrorPreferencePage.class);
+
+	/**
+	 * Construct the preference page with "Maven" > "Errors/Warnings".
+	 */
+	public MavenErrorPreferencePage() {
+		super(new String[] { "Maven", "Errors/Warnings" });
+	}
+	
+	/**
+	 * Getter method for "groupId duplicate of parent groupId" LabeledCombo
+	 * @return String value of selected item in labeled combo
+	 */	
+	public String getGroupId () {
+		return new LabeledCombo(GROUP_ID).getSelection();
+	}
+	
+	/**
+	 * Setter method for "groupId duplicate of parent groupId" LabeledCombo
+	 * @param groupId
+	 */
+	public void setGroupId (String groupId) {
+		new LabeledCombo(GROUP_ID).setSelection(groupId);
+		this.apply();
+		log.info("Set Maven error/warning Group ID to: " + groupId);
```

Comment:
In all setters in this class use constant, which is already created, instead of string of a label. If a label would be changed in a next Eclipse release, we could forget on changing of this, but with existing constants it will not.